cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -19,7 +19,7 @@ function deploy_kwok() {
 function copy_istio_cni_conf_files() {
     if [ "$KUBEVIRT_DEPLOY_ISTIO" == "true" ] && [ "$KUBEVIRT_WITH_CNAO" == "true" ]; then
         for nodeNum in $(seq -f "%02g" 1 $KUBEVIRT_NUM_NODES); do
-            $ssh node${nodeNum} -- "until ls /etc/cni/multus > /dev/null 2>&1; do sleep 1; done"
+            timeout 60 $ssh node${nodeNum} -- "until ls /etc/cni/multus > /dev/null 2>&1; do sleep 1; done"
```

Comment:
This change is not related to this PR according to title and description.

Please move this change to new PR.

Reply:
Indeed, but it was added as a part of a previous review: https://github.com/kubevirt/kubevirtci/pull/1262#discussion_r1740808214

Reply:
Nevertheless, this change is not related to this PR, right?

Reply:
Yes its not, maybe we can separate it another PR indeed

Reply:
@aerosouund Let's minimize the changes so we fix the issues first and then we will improve